app/views/users/show.erb

```diff
@@ -2,7 +2,7 @@
 
 <h2>Reservation history:</h2>
 <ul>
-<% Booking.where(user_id: session[:id]).order(:time).each { |booking| %>
+<% Booking.where(user_id: session[:user_id]).order(:time).each { |booking| %>
```

Comment:
you should be using `current_user.bookings.order(:time)` here instead. Always try to not use your session in a view.